scripts/main.js

```diff
@@ -0,0 +1,118 @@
+let header = document.querySelector(".main-banner");
+let headerSlides = ['url("images/home-bg.jpg")', 'url("images/home-bg-2.jpg")']
+let toggleButton = document.querySelector(".toggle-button");
+let navBar = document.querySelector(".main-nav__mobile");
+let desktopItems = document.querySelectorAll(".main-nav__item");
+let mobileItems = document.querySelectorAll(".main-nav__mobile-items");
+let questions = document.querySelectorAll(".question-item");
+
+function showHomeSlides(index = 0) {
+    header.style.backgroundImage = headerSlides[index];
+    if (index === 1) {
+        setTimeout(() => showHomeSlides(0), 4000);
+    } else {
+        setTimeout(() => showHomeSlides(1), 4000);
```

Comment:
this is creating multiple timeouts, and they are not cleared, remove the timeouts before adding new ones